### ISSUE TITLE
Latest version often fails to sync

### ISSUE THREAD

**Issue by ghgary** [mpcjanssen/simpletask-android#1092]
For the last few weeks, since the last update, I think, SimpleTask often fails to read updates to todo.txt made by Dropbox. I am running version 10.5.5 on a Samsung Galaxy running Android 9.

I often update todo.txt on a computer running Linux. I can see the changes to todo.txt on my phone using the Dropbox editor, so Dropbox appears to be working fine. However, SimpleTask shows a previous version of the file. Executing sync has no effect. I can only fix the problem by adding an entry to SimpleTask and saving it, then reconciling the 'conflicted copy" that appears in my laptop's Dropbox/todo directory.

**Comment by mpcjanssen** [comment]
Doe the new version get picked up if you re-open the file from the menu in Simpletask using `Open task list file` ?

**Comment by ghgary** [comment]
Yes, that works. Also, since using that command to open the file, the syncing problem seems to have disappeared. Changes saved from Simpletask appear as they should on my Linux machine and changes made to todo.txt on Linux appear shortly in Simpletask without my having to do anything. So, apparently something was stuck? In any case, thanks for the fix.

**Comment by kianboon80** [comment]
> Doe the new version get picked up if you re-open the file from the menu in Simpletask using `Open task list file` ?

For my case, it doesn't get reload when I reopen file from task. Neither does it sync. I just switch phone to android 10. It is still working on on my old phone on android 9. Any workaround?
